lib/plugins/Label_24_Slash.test.ts

```diff
@@ -0,0 +1,48 @@
+import { MessageDecoder } from '../MessageDecoder';
+import { Label_24_Slash } from './Label_24_Slash';
+
+describe('Label_24_Slash', () => {
+  let plugin: Label_24_Slash;
+
+  beforeEach(() => {
+    const decoder = new MessageDecoder();
+    plugin = new Label_24_Slash(decoder);
+  });
+
+  test('matches qualifiers', () => {
+    expect(plugin.decode).toBeDefined();
+    expect(plugin.name).toBe('label-24-slash');
+    expect(plugin.qualifiers).toBeDefined();
+    expect(plugin.qualifiers()).toEqual({
+      labels: ['24'],
+      preambles: ['/'],
+    });
+  });
+
+
+  test('valid', () => {
+    // https://app.airframes.io/messages/3439806391
+    const text = '/241710/1021/04WM/34962/N53.13/E001.33/3374/1056/';
+    const decodeResult = plugin.decode({ text: text });
+    expect(decodeResult.decoded).toBe(true);
+    expect(decodeResult.decoder.decodeLevel).toBe('partial');
+    expect(decodeResult.raw.message_timestamp).toBe(1729160460);
+    expect(decodeResult.raw.flight_number).toBe('04WM');
+    expect(decodeResult.raw.altitude).toBe(34962);
+    expect(decodeResult.raw.position.latitude).toBe(53.13);
+    expect(decodeResult.raw.position.longitude).toBe(1.33);
+    expect(decodeResult.raw.eta_time).toBe(39360);
+    expect(decodeResult.formatted.items.length).toBe(3);
```

Comment:
went a little different with the specs - assuming the formatted will be fine. not sure how i feel about it

Reply:
Interesting. I suppose it makes the spec simpler, but I suppose we won't know that formatted results have changed accidentally.

Reply:
Added until there's specific tests for ResultFormatter